server/interfaces.coffee

```diff
@@ -33,3 +34,15 @@ Meteor.methods
         dateUTC = match[1].replace(' ', 'T') + offset
         dateUTC
     )
+
+  queryForSuggestedArticles: (eventId) ->
+    event = grid.UserEvents.findOne(eventId)
+    console.log "Calling SPA API @ " + SPA_API_URL
+    response = HTTP.call('GET', "#{SPA_API_URL}/relatedArticles", {
+      params: { eventName: event.summary }
```

Comment:
I would like the API for this to be more generic. We could use a "/find" endpoint that takes arbitrary mongo queries as a parameter. That way all the EIDR specific code for building the query from an EIDR event would stay in the EIDR codebase.